src/site/stages/build/process-cms-exports/transformers/media-image.js

```diff
@@ -8,7 +8,16 @@ const transform = entity => ({
   entityMetatags: createMetaTagArray(entity.metatag.value),
   fieldMediaInLibrary: getDrupalValue(entity.fieldMediaInLibrary),
   fieldOwner: entity.fieldOwner[0],
-  image: entity.image[0],
+  image: {
+    alt: entity.image[0].alt || '',
+    title: entity.image[0].title || '',
+    url: entity.thumbnail[0].url,
+    derivative: {
+      url: entity.thumbnail[0].url,
+      width: entity.image[0].width,
+      height: entity.image[0].height,
+    },
```

Comment:
I expect this is going to change once we figure out how to use the `crop`s, but we'll cross that bridge shortly.

Reply:
Right, this will most likely need to be updated once we learn how `crops` work.